Angular/package.json

```diff
@@ -34,7 +34,7 @@
     "@angular/compiler-cli": "^15.1.0",
     "@types/jasmine": "~4.3.0",
     "eslint": "^8.39.0",
-    "eslint-config-devextreme": "1.1.2",
+    "eslint-config-devextreme": "^1.1.3",
```

Comment:
```suggestion
    "eslint-config-devextreme": "1.1.3",
```

Reply:
Eup, agree with you.
I missed it :)

Reply:
marking as unresolved, since other frameworks also have "^"

Reply:
Yup, I didn't want to resolve it with GitHub :)
I fixed it in the examples-template repo and merge these changes to this PR